geometry_msgs/Point32[] particle_states
std_msgs/Float32[] particle_weights
geometry_msgs/Point32 estimated_state
